index.js

```diff
@@ -83,7 +148,21 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+if(movies.length === 0){
+  return null;
+}
+let movieObj = [];
+for (let movie of movies){
+  if(movie.imdbID === id){
+    movieObj.push(movie.imdbID);
```

Comment:
Here we should just `return movie` and break out of our function.  The way we have our function running currently, in the event that we do not have a match we set movieObj to null.  This means that _even if we find a match_ the next iteration _doesn't  pass your conditional on line 157 so we set movieObj to null_